### ISSUE TITLE
JOGL 2.4

### ISSUE THREAD

**Issue by processing-bot** [processing/processing4#84]
*Created by: sampottinger*

It looks like the 2.4 train has reached some viable release candidates. Might be time to make the jump! Just creating this issue to track some of the efforts :)

**Comment by processing-bot** [comment]
*Created by: sampottinger*

Blocked on #73 .

**Comment by processing-bot** [comment]
*Created by: sampottinger*

Blocked on #76

**Comment by processing-bot** [comment]
*Created by: benfry*

@sampottinger is your plan to leave this open for the final 2.4, or was it just a note about updating the RC?

**Comment by processing-bot** [comment]
*Created by: sampottinger*

I'll close this and open another for a future version.

**Comment by processing-bot** [comment]
*Created by: github-actions[bot]*

This issue has been automatically locked since there has not been any recent activity after it was closed. Please open a new issue for related bugs.
